blockmanager_test.go

```diff
@@ -360,26 +382,37 @@ func TestBlockManagerInitialInterval(t *testing.T) {
 				r := *responses[index]
 
 				// Let the blockmanager handle the message.
-				if !f(nil, msgs[index], responses[index]) {
+				done, _ := queries[index].HandleResp(
+					msgs[index], &r, "",
+				)
+
+				if !done {
 					t.Fatalf("got response false on "+
 						"send of index %d: %v",
 						index, testDesc)
 				}
 
+				continue
```

Comment:
Stray continue?

Reply:
Yep, leftover needed to make tests pass at one point. Fixed here: https://github.com/lightninglabs/neutrino/pull/179/commits/dab9f9f4046ea7711cbe1481ef4f52f7552f38f3